### ISSUE TITLE
TVDB v3 API doesn't work with 16 character API key

### ISSUE THREAD

**Issue by cjzzz** [mynttt/UpdateTool#62]
I have a legacy API key for TVDB which is 16 characters long. If I try to use it with this tool on the latest version, I get the following error:

```
[INFO ] - 2021-05-13 18:30:52 @ TvdbApiV4.auth: Testing TMDB API (v4) pin: xxxxxxxxxxxxxxxx
[ERROR] - 2021-05-13 18:30:53 @ TvdbApiV4.auth: TVDB v4 authorization failed with code 401 and status failure
[ERROR] - 2021-05-13 18:30:53 @ TvdbApiV4.auth: This could be due to the TVDB API v4 having issues at the moment or your credentials being wrong.
[ERROR] - 2021-05-13 18:30:53 @ TvdbApiV4.auth: This is the received response:
[ERROR] - 2021-05-13 18:30:53 @ TvdbApiV4.auth: Payload:{"status":"failure","message":"InvalidAPIKey: pin invalid","data":null}
```

This appears to be because only keys that are at least 32 characters are treated as v3 keys.

I can successfully authenticate with the v3 TVDB API manually [(https://api.thetvdb.com/swagger)](https://api.thetvdb.com/swagger) using the just the API key.

**Comment by mynttt** [comment]
I'm surprised that the legacy keys can have other formats as well. I'll add a patch that 16 character keys are regognized as well!

**Comment by mynttt** [comment]
Fixed with 1.5.6

**Comment by cjzzz** [comment]
Thanks @mynttt 

It looks like the initial check now works, but the process fails later due to the `isTvdbV4` property of `ImdbPipeline.java`:

```
[INFO ] - 2021-05-17 11:42:08 @ TvdbApiV3.<init>: Testing TVDB API (v3) authorization apikey: xxxxxxxxxxxxxxxx
[INFO ] - 2021-05-17 11:42:09 @ TvdbApiV3.<init>: Test passed. API Key is valid.
...
[INFO ] - 2021-05-17 11:42:11 @ ImdbDockerImplementation$ImdbBatchJob.run: Library IDs on ignore list: []
[INFO ] - 2021-05-17 11:42:11 @ ImdbRatingDatasetFactory.readData: Reading data...
[INFO ] - 2021-05-17 11:42:13 @ ImdbRatingDatasetFactory.readData: 1151933 lines read.
[INFO ] - 2021-05-17 11:42:13 @ TvdbApiV4.auth: Testing TMDB API (v4) pin: xxxxxxxxxxxxxxxx
[ERROR] - 2021-05-17 11:42:14 @ TvdbApiV4.auth: TVDB v4 authorization failed with code 401 and status failure
[ERROR] - 2021-05-17 11:42:14 @ TvdbApiV4.auth: This could be due to the TVDB API v4 having issues at the moment or your credentials being wrong.
```

**Comment by mynttt** [comment]
Good find! Seems like I have two checks I'll update that as well later!

**Comment by mynttt** [comment]
Has been fixed with 1.5.6b!
